index.html

```diff
@@ -0,0 +1,166 @@
+                <h3 class="course_name">Html</h3>
+                <p class="course_info">Get started as a web developer by learning the basics of HTML, the essential
+                    language of the web.</p>
+            </div>
+            <div class="card card2 d_flex_row">
+                <img class="course_icon" src="./images/css.png" alt="css">
+                <h3 class="course_name">CSS</h3>
+                <p class="course_info">Learn the fundamentals of CSS to add beautiful styling and customize the
+                    appearance of your web pages.</p>
+            </div>
+            <div class="card card3 d_flex_row">
+                <img class="course_icon" src="./images/javascript.png" alt="javascript">
+                <h3 class="course_name">Javascript</h3>
+                <p class="course_info">JavaScript is among the most powerful and flexible programming languages of
+                    the web.</p>
+            </div>
+            <div class="card card4 d_flex_row">
+                <img class="course_icon" src="./images/atom.png" alt="react">
+                <h3 class="course_name">React</h3>
+                <p class="course_info">Learn to build dynamic and powerful web apps using React.js, a
+                    component-based front-end framework.</p>
+            </div>
+            <div class="card card5 d_flex_row">
+                <img class="course_icon" src="./images/sql.png" alt="SQL">
+                <h3 class="course_name">SQL</h3>
+                <p class="course_info">Learn to communicate with databases using SQL, the standard data management
+                    language.</p>
+            </div>
+            <a class="course_link" href="#">Join The Courses Now</a>
+        </div>
+    </section>
+    <section class="teaching_team">
+        <div class="section_headline d_flex_column">
+            <h2 class="section_title">Teaching Team</h2>
+            <hr>
+        </div>
+        <div class="the_team">
+            
+        </div>
+          <button class="more_btn d_flex_row">
+            <span class="btn_text">MORE</span>
+            <img class="arrow" src="./images/arrow-down.png" alt="arrow down">
+          </button>
+    </section>
+    <Footer>
+        <div class="partners_section">
+            <div class="section_title d_flex_column">
+                <h2 class="title footer_title">Partner</h2>
+                <hr>
+            </div>
+            <ul class="partners d_flex_row">
+                <li><img class="partner_img" src="./images/google.png" alt="google"></li>
+                <li><img class="partner_img" src="./images/ted.png" alt="ted"></li>
+                <li><img class="partner_img" src="./images/blogger.png" alt="blogger"></li>
+                <li><img class="partner_img" src="./images/windows-logo.png" alt="windows-logo"></li>
+                <li><img class="partner_img" src="./images/github.png" alt="github"></li>
```

Comment:
- [ ] [OPTIONAL] I also noticed the sizing of the image are a bit large. I suggest we try to reduce the sizes here as well.